### PR TITLE
Removing authorization from keycloak

### PR DESCRIPTION
**Pull request by emiteze**

### CODE REVIEW

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/WorkspacePermissions.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package io.charlescd.moove.domain
```
Comment (eduardomiyoshizup) [inline moove/domain/src/main/kotlin/io/charlescd/moove/domain/WorkspacePermissions.kt:1]: Missing license header

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/WorkspaceStatusEnum.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package io.charlescd.moove.domain
```
Comment (eduardomiyoshizup) [inline moove/domain/src/main/kotlin/io/charlescd/moove/domain/WorkspaceStatusEnum.kt:1]: Missing license header

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/WorkspacePermissions.kt (hunk 1 above)
Comment (emiteze) [reply moove/domain/src/main/kotlin/io/charlescd/moove/domain/WorkspacePermissions.kt:17]: Fixed

File: moove/domain/src/main/kotlin/io/charlescd/moove/domain/WorkspaceStatusEnum.kt (hunk 2 above)
Comment (emiteze) [reply moove/domain/src/main/kotlin/io/charlescd/moove/domain/WorkspaceStatusEnum.kt:17]: Fixed

**Review by pedroborgeszup — APPROVED** [review]

**Review by monicaribeiro — APPROVED** [review]

**Pull request merged**